aws-sqs-queue/outputs.tf

```diff
@@ -0,0 +1,23 @@
+output "normal_queue_name" {
+  value = aws_sqs_queue.normal.name
+}
```

Comment:
😳 co takhle taději nechat jenom `queue_name` a tu resorusu pojmenovat jen `queue` nebo `this`? Ten normal mi zní tak nějak podivně. 😅

Reply:
Joo, dava smysl 😅

Reply:
Upraveno